README.rst

```diff
@@ -57,8 +59,7 @@ Initialization
         Define you target on which you would like to evaluate
         the featureFlag
         """
-        target = Target(identifier="user1")
-        target = Target(name="User1")
+        target = Target(identifier="user1",account="my-acc",org="my-org",environment="my-env",project="my-project",name="user1")
```

Comment:
Target from auth_target.py should be used here. The target used from api spec should be refactorized and returned to the previous state corresponding to auth_target